src/cmd/node/rpc/get_module_abi.rs

```diff
@@ -3,6 +3,7 @@ use clap::Parser;
 use jsonrpsee::core::client::ClientT;
 use jsonrpsee::http_client::HttpClientBuilder;
 use jsonrpsee::rpc_params;
+use move_vm_backend::abi::ModuleAbi;
```

Comment:
Let's move this `ModuleAbi` to `move_vm_backend_common` and then import it from there to keep `smove` free from all the heavy stuff within the `move_vm_backend`